Cache/Requests.lean

```diff
@@ -8,6 +8,29 @@ import Cache.Hashing
 
 namespace Cache.Requests
 
+/-- Attempts to determine the running project's GitHub repository from its `origin` Git remote. -/
+def getRemoteRepo : IO String := do
+  let out ← IO.Process.output {cmd := "git", args := #["remote", "get-url", "origin"]}
```

Comment:
Shouldn't we be looking at the tracking remote of the current branch, rather than assuming `origin`?

Reply:
Does lake use it even if `clone.defaultRemoteName` is set to something else in the global git config?